google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java

```diff
@@ -0,0 +1,126 @@
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+  }
+
+  public static void main(String[] args) throws IOException {
+    UploadMediaBundleParams params = new UploadMediaBundleParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new UploadMediaBundle().runExample(googleAdsClient, params.customerId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId) throws IOException {
+
+    byte[] bundleData = ByteStreams.toByteArray(new URL(BUNDLE_URL).openStream());
+
+    // Creates a media bundle file.
+    MediaBundle bundle =
+        MediaBundle.newBuilder().setData(BytesValue.of(ByteString.copyFrom(bundleData))).build();
+
+    MediaFile file =
+        MediaFile.newBuilder()
+            .setName(StringValue.of("Ad Media Bundle"))
+            .setType(MediaType.MEDIA_BUNDLE)
+            .setSourceUrl(StringValue.of(BUNDLE_URL))
+            .setMediaBundle(bundle)
+            .build();
+
+    // Creates a MediaFile operation.
+    MediaFileOperation op = MediaFileOperation.newBuilder().setCreate(file).build();
+
+    // Creates the MediaFile service client.
+    try (MediaFileServiceClient mediaFileServiceClient =
+        googleAdsClient.getLatestVersion().createMediaFileServiceClient()) {
+      MutateMediaFilesResponse response =
```

Comment:
Add a line comment explaining this sends a mutate operation to add the bundle, or just change the comment on line 116 to match the [PHP example](https://github.com/googleads/google-ads-php/blob/e1ff0e4aa64d6c640a76300e8a000dd2486da180/examples/Misc/UploadMediaBundle.php#L114).

Reply:
Sounds good.